src/sign/sign.ts

```diff
@@ -1,27 +1,43 @@
-import { webcrypto } from 'crypto';
-import { SIGNING_ALGORITHM_CONFIG, isSigningAlgorithm } from '../algorithms';
+const webcrypto = globalThis.crypto;
```

Comment:
curious if we could improve the file structure/name - the double index on `sign`